projects/media-viewer/src/lib/toolbar/toolbar-button-visibility.service.ts

```diff
@@ -30,7 +31,8 @@ export class ToolbarButtonVisibilityService {
     this.showNavigation = false;
     this.showZoom = false;
     this.showRotate = false;
-    this.showPresentationMode = false;
+    this.showPresentationMode = true;
```

Comment:
options should not be set to true here, because this method is resetting all the options, so that they can be set with the options object being passed in